parcels/examples/tutorial_geospatial.ipynb

```diff
@@ -0,0 +1,1056 @@
+    "\n",
+    "        placemark = etree.SubElement(document, \"Placemark\")\n",
+    "        name_element = etree.SubElement(placemark, \"name\")\n",
+    "        name_element.text = str(trajectory_idx)\n",
+    "        \n",
+    "        # Link custom icon styling\n",
+    "        if rubber_ducks:\n",
+    "            style_url = etree.SubElement(placemark, \"styleUrl\")\n",
+    "            style_url.text = \"#iconStyle\"\n",
+    "        \n",
+    "        gx_track = etree.SubElement(placemark, \"{%s}Track\" % gx_ns)\n",
+    "        altitude_mode = etree.SubElement(gx_track, \"{%s}altitudeMode\" % gx_ns, text=\"clampToGround\")\n",
+    "\n",
+    "        for time in trajectory_gdf[\"time\"]:\n",
+    "            when_element = etree.SubElement(gx_track, \"when\")\n",
+    "            when_element.text = time.strftime(\"%Y-%m-%dT%H:%M:%SZ\")\n",
+    "\n",
+    "        for _, row in trajectory_gdf.iterrows():\n",
+    "            gx_coord_element = etree.SubElement(gx_track, \"{%s}coord\" % gx_ns)       \n",
+    "            gx_coord_element.text = f\"{row['geometry'].x} {row['geometry'].y} 0\"\n",
+    "        \n",
+    "    # Save the KML to a file\n",
+    "    with open(path, \"wb\") as f:\n",
+    "        f.write(etree.tostring(kml_out, pretty_print=True))\n",
+    "    return\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "parcels_geopandas_to_kml(gdf_parcels, DATA_OUTPUT_FOLDER / f\"{DATA_OUTPUT_NAME}.kml\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Opening this kml file in Google Earth, we can explore the particle trajectories interactively:\n",
+    "![Rubber ducks in the ocean](tutorial_geospatial_google_earth.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Kepler.gl\n",
+    "> [Kepler.gl](https://kepler.gl/) is a data-agnostic, high-performance web-based application for visual exploration of large-scale geolocation data sets. Built on top of Mapbox GL and deck.gl, kepler.gl can render millions of points representing thousands of trips and perform spatial aggregations on the fly  \n",
+    "> - [Kepler Docs](https://docs.kepler.gl/)\n",
+    "\n",
+    "For smaller visualisations, using the Kepler demo web interface is enough. To animate trips/trajectories in the web interface some customisation to the geojson file is required.\n",
+    "\n",
+    "> In order to animate the path, the geoJSON data needs to contain `LineString` in its feature geometry, and the coordinates in the LineString need to have 4 elements in the formats of `[longitude, latitude, altitude, timestamp]` with the last element being a timestamp. Valid timestamp formats include unix in seconds such as `1564184363` or in milliseconds such as `1564184363000`.\n",
+    "> - Kepler.gl tooltip\n",
```

Comment:
Should this be a link?

Reply:
The tooltip is part of the web app, and doesn't have a link associated with it.